app/assets/javascripts/admin/team/team_list_view.js

```diff
@@ -132,6 +142,7 @@ class TeamListView extends React.PureComponent<Props, State> {
                 defaultPageSize: 50,
               }}
               style={{ marginTop: 30, marginBotton: 30 }}
+              locale={{ emptyText: this.renderPlaceholder() }}
```

Comment:
Isn't there always a default organization team? So, the table will never be empty? At least, I couldn't make the UI to show this.

Reply:
I wasn't sure about that. If that is the case, I should probably show this placeholder when there is only one team?

Reply:
That’s true, currently, the organization team is automatically created and cannot be deleted